src/inventory_ui.h

```diff
@@ -430,7 +430,7 @@ class inventory_column
         bool paging_is_valid = false;
         bool visibility = true;
 
-        size_t selected_index = 0;
+        size_t selected_index = -1;
```

Comment:
Looks weird to assign `-1` to a `size_t` variable 🤣

Reply:
It just means assigning the max value of the unsigned int type. If I'm not mistaken it's well-defined behavior according to the c++ standard.

Reply:
better use `std::numeric_limits<size_t>::max()` or add a comment that makes it clear the overflow is intentional